best_fit_line.py

```diff
@@ -39,12 +39,12 @@
     prob += (a*x - y - c >= -z)
 
 status = prob.solve(GLPK(msg = 0))
-print "status:", LpStatus[status]
-print "values:"
-print "\ta:", value(a)
-# print "\tb:", value(b)
-print "\tc:", value(c)
-print "\tz:", value(z)
+print("status:", LpStatus[status])
```

Comment:
Please add "__from \_\_future\_\_ import print_function__" as the first line of this script or this line will print a tuple on Python 2.